Gemfile

```diff
@@ -5,6 +5,7 @@ gem 'spree', github: 'spree/spree', branch: 'main'
 gem 'spree_backend', github: 'spree/spree_backend', branch: 'main'
 gem 'spree_frontend', github: 'spree/spree_legacy_frontend', branch: 'main'
 gem 'spree_emails', github: 'spree/spree', branch: 'main'
+gem 'rspec_junit_formatter', '~> 0.4.1'
```

Comment:
@aleksandarpetrushev let's add this please to `sptee_dev_tools` repo - https://github.com/spree/spree_dev_tools

Reply:
Added https://github.com/spree/spree_dev_tools/commit/f4bb93916ec9e80379fae6e5aeffddf60dbb02cc